paasta_tools/setup_istio_mesh.py

```diff
@@ -262,8 +279,10 @@ def main() -> None:
     kube_client = KubeClient()
     ensure_namespace(kube_client, namespace="paasta")
 
+    global RATE_LIMIT
+    RATE_LIMIT = args.rate_limit
```

Comment:
just curious: how come we're using a global here rather than plumbing `args.rate_limit` through `process_kube_services` and then to all the other rate-limited functions?

Reply:
Since rate_limit is a read-only variable and passing around rate_limit and api_updates to each function quickly became messy. Initially I thought about making rate_limit global but we have dropped that idea. https://github.com/Yelp/paasta/pull/3201 Have details on how we are planning to handle rate limiting